bp_me/syn/Makefile

```diff
@@ -85,19 +85,19 @@ lint.me:
 	@echo "ME lint not supported"
 
 regress.me.sc: dirs.sc
-	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P)
-	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P) SKIP_INIT_P=1
-	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce_fsm NUM_INSTR_P=$(NUM_INSTR_P)
-	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce_fsm NUM_INSTR_P=$(NUM_INSTR_P) SKIP_INIT_P=1
+	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P) PROG=normal
+	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce NUM_INSTR_P=$(NUM_INSTR_P) SKIP_INIT_P=1 PROG=uncached
+	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce_fsm NUM_INSTR_P=$(NUM_INSTR_P) PROG=normal
+	$(MAKE) -j 1 build.sc sim.sc TB=bp_cce_fsm NUM_INSTR_P=$(NUM_INSTR_P) SKIP_INIT_P=1 PROG=uncached
```

Comment:
FYI, setting the program here to some dummy values so that unique logs are generated for all tests. If PROG is not specified, then the SKIP_INIT_P=1 tests overwrite the other one for the same config, potentially hiding a failure.

Since we use CFG and PROG to generate unique test result paths, and don't use the value of any specific params, setting PROG is the easiest way to get a unique name per test.

Reply:
Doesn’t TB affect the path though?

Reply:
yes, but the ME testbenches have a parameter that determines if they run in cached/normal mode or uncached mode. (This param gets passed to the trace rom generator to generate either cached accesses or uncached accesses). But this param doesn't affect path.

So the second test (for the same TB setting) overwrites the first, unless PROG is set to differentiate.

Reply:
gotcha!